docs/topics/release-notes.md

```diff
@@ -24,6 +24,12 @@ The timeline for deprecation of a feature present in version 1.0 would work as f
 
 * Version 1.3 would remove the deprecated bits of API entirely.
 
+Deprecations are marked using `rest_framework.compat.deprecated`, which accepts a version tuple for the version when code is first deprecated and message to pass to the `warnings` module:
+
+    from rest_framework.compat import deprecated
+    ...
+    deprecated((3,1,0), "Using X for Y is deprecated. Prefer Z")
```

Comment:
I used a three value tuple because people are used to those. eg. `3.2.0 => (3,2,0)` — but you're right 't's not needed.